recipe/conda_build_config.yaml

```diff
@@ -3,6 +3,15 @@ c_compiler:          # [osx]
 cxx_compiler:        # [osx]
   - clang_bootstrap  # [osx]
 
+# clang 17 should remain compatible with libcxx 17 (which will be pulled in
+# as a test dependency despite requiring `__osx>=10.13`). Since libcxx has
+# removed workarounds for MacOS<10.13, clang will fail while including
+# the C++ headers, because those rely on things not present in older SDKs.
+MACOSX_SDK_VERSION:         # [osx and x86_64]
+  - "10.13"                 # [osx and x86_64]
+MACOSX_DEPLOYMENT_TARGET:   # [osx and x86_64]
+  - "10.13"                 # [osx and x86_64]
+
```

Comment:
The comment tries to lay out my understanding. It was necessary during the rc-phase once there was a libcxx 17, which would get pulled in by clang, which would then fail when including the C++ stdlib (because how libcxx has removed fallbacks for the old macos SDK).

In theory, the requirement `__osx>=10.13` on libcxx should be enough to exclude it in this feedstock, but that's not how our CI seems to work. So I needed to bump to 10.13 here as well. I didn't distinguish between SDK and deployment target, because I thought it would be easier on the resolver if the compiler itself also has the requirement, rather than just the stdlib (which is why I had also added it to the compiler activation).

I'm fine to revert this for now if you prefer. We'll just have to re-fix this feedstock one way or another as soon as libcxx 17 is live.

Reply:
Let's remove it for now. Let's deal with it when I see an actual error.